src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -407,6 +407,10 @@ $color-info-bubble-text: #818a8d;
   .facility-detail {
     padding: 0 1.5rem;
 
+    ul.sublist {
```

Comment:
I think you can use the padding utility from the design system for this. Adding `vads-u-padding-left--0` to the className should achieve the same effect. More info here: https://design.va.gov/utilities/padding#padding-left

Reply:
The less styling we introduce the better! Thanks! :)